plugins/modules/kubevirt_cdi_upload.py

```diff
@@ -11,7 +11,7 @@
 
 DOCUMENTATION = '''
 
-module: kubevirt_cdi_upload
+module: community.kubevirt.kubevirt_cdi_upload
```

Comment:
short name here

Reply:
@gundalow thanks! I guess I was confused by the wording of the first bullet point here: https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#documentation

I'll think of an improvement and send a patch.

Reply:
Perfect, that's exactly the type of thing we want to improve.